analyzers/tests/SonarAnalyzer.Test/TestCases/AspNet/AvoidUnderPosting.CSharp12.cs

```diff
@@ -8,8 +8,15 @@ public class ModelWithPrimaryConstructor(int vp, int rvp, int nvp)
         public int ValueProperty { get; set; } = vp;                        // Compliant - no parameterless constructor, type cannot be used for Model Binding
     }
 
+    public class ModelWithPrimaryAndParameterlessConstructor(int vp, int rvp, int nvp)
+    {
+        public ModelWithPrimaryAndParameterlessConstructor() : this(0, 0, 0) { }
+        public int ValueProperty { get; set; } = vp;                        // Noncompliant
+    }
+
```

Comment:
Educational: Is this meant just to use `Verify`?

Reply:
It is there to demonstrate that the rule works with primary constructors if an additional parameter less constructor exists (and it allows me to use `.Verify`).